docs/UserGuide.adoc

```diff
@@ -156,6 +156,11 @@ Views all details of the 1st person in the results of the `find` command.
 Clears all entries from the address book. +
 Format: `clear`
 
+== Sorting the entries : `sort`
+
+Sort the entries by name in alphabetical order (ASC) +
```

Comment:
Minor nit but the description should be aligned with others (i.e. `Sorts the entries...`, full stop)